app/views/landing_page/_search_filter.html.erb

```diff
@@ -0,0 +1,55 @@
+<%= form_for @form, url: search_path, html: { class: 'form' }, builder: Dough::Forms::Builders::Validation do |f| %>
```

Comment:
This partial shouldn't be around anymore. I think there has been a rebase boo boo at some point down the line.